cwms-data-api/src/main/java/cwms/cda/data/dto/basin/Basin.java

```diff
@@ -0,0 +1,195 @@
+package cwms.cda.data.dto.basin;
+
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTOBase;
+import cwms.cda.data.dto.basinconnectivity.Stream;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.NamedPgJsonFormatter;
+import cwms.cda.formatters.json.PgJsonFormatter;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+@FormattableWith(contentType = Formats.NAMED_PGJSON, formatter = NamedPgJsonFormatter.class, aliases = {Formats.DEFAULT, Formats.JSON})
+@FormattableWith(contentType = Formats.PGJSON, formatter = PgJsonFormatter.class)
+public final class Basin implements CwmsDTOBase {
```

Comment:
We want to remove the association of the Basin DTO to the Stream DTO here. 

The streams should just be the stream Id, and same with the sub-basins (should just the the IDs)... those probably should be LocationIdentifier objects actually. 

If we keep this association, then we are forced to query those objects like the basin-connectivity endpoint does already, which we want to avoid, since we think it should be the client that handles that business.

Reply:
After looking at the sql code, just remove the streams and sub-basins fields from this DTO

Reply:
Addressed changes